yarn.lock

```diff
@@ -1294,6 +1299,18 @@
     debug "^4.1.1"
     utf8 "3.0.0"
 
+"@celo/connect@1.0.2":
+  version "1.0.2"
+  resolved "https://registry.yarnpkg.com/@celo/connect/-/connect-1.0.2.tgz#8a53be74243c10a6990a19e1cf10b68c34afdb12"
+  integrity sha512-kCeOmSU+f+ufb8G2VU2UnOS28QznooBUt5Vgzvg7+ZoUmTB7PBxqPSw4ViQdRxSAMi1yZihTx/HX9PijZ1Mptw==
+  dependencies:
+    "@celo/utils" "1.0.2"
+    "@types/debug" "^4.1.5"
+    "@types/utf8" "^2.1.6"
+    bignumber.js "^9.0.0"
+    debug "^4.1.1"
+    utf8 "3.0.0"
+
 "@celo/contractkit@1.0.1":
```

Comment:
Looks like another package still references `contractkit@1.0.1`, would be nice to upgrade them all.